package.json

```diff
@@ -8,7 +8,8 @@
     "build": "tsc && vite build",
     "preview": "vite preview",
     "lint": "eslint src",
-    "format": "prettier --write . && yarn lint --fix",
+    "prettier": "prettier --check src",
```

Comment:
We don't have a separate script for this in other packages. It's a sort of lint so we have "lint" do it. Consider aligning with other repos.

Reply:
Done